lisa/sut_orchestrator/azure/tools.py

```diff
@@ -579,3 +580,91 @@ def get_pool_records(self, pool_id: int, force_run: bool = False) -> Dict[str, s
             records[content_split[i]] = content_split[i + 1]
 
         return records
+
+
+class BlobDownloader(Tool):
+    @property
+    def command(self) -> str:
+        return "echo blob_downloader"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def download_blob(
+        self,
+        account_name: str,
+        container_name: str,
+        blob_name: str,
+        credential: Optional[Any] = None,
+        subscription_id: Optional[str] = None,
+        cloud: Optional[Cloud] = None,
+        resource_group_name: Optional[str] = None,
+        connection_string: Optional[str] = None,
+    ) -> PurePath:
+        from lisa import constants
```

Comment:
Move them to top, avoid to import in methods.

Reply:
if move them to the top, it will have a circular import error.

Reply:
Try import constants from `lisa.util`

Reply:
it works. but `from lisa.sut_orchestrator.azure.common import get_or_create_storage_container` still can't be moved to the top.

Reply:
I don't see cycle imports from common to other azure modules. What's error here?

Reply:
now, it worked. updated